packages/cardpay-sdk/sdk/utils/signing-utils.ts

```diff
@@ -435,3 +435,15 @@ function sortSignaturesAsBytes(
     return [contractSignature, ownerSignature];
   }
 }
+
+export function signatureToVRS(rawSignature: string) {
+  const signature = strip0x(rawSignature);
+  const v = parseInt(signature.substr(64 * 2));
+  const r = signature.substr(0, 32 * 2);
+  const s = signature.substr(32 * 2, 32 * 2);
+  return { v, r, s };
+}
+
+export function strip0x(s: string) {
+  return Web3.utils.isHexStrict(s) ? s.substr(2) : s;
```

Comment:
using `s.replace(/^0x/, '')` is better since it's idempotent--you can run it as many times as you want and it will always return the correct thing.

Reply:
Agree! Will change it